src/patterns/equality-information/disability-impact/index.njk

```diff
@@ -0,0 +1,47 @@
+---
+title: Disability impact equality question
+layout: layout-example.njk
+---
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{{ govukRadios({
+  idPrefix: "disability",
+  name: "disability",
+  fieldset: {
+    legend: {
+      text: "Do any of your conditions or illnesses reduce your ability to carry out day to day activities?",
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  hint: {
+    text: "For example eating, washing, walking or going shopping."
+  },
+  items: [
+    {
+      value: "little",
+      text: "Yes, a little"
+    },
+    {
+      value: "lot",
+      text: "Yes, a lot"
+    },
+    {
+      value: "No",
+      text: "Not at all"
+    },
```

Comment:
Added to the draft doc as well:

> The order of these seems a little jumbled – it would make more sense to order by severity and go 'Yes, a lot', 'Yes, a little', 'Not at all'.